zhaquirks/tuya/ts0601_dimmer.py

```diff
@@ -148,6 +148,7 @@ class TuyaSingleSwitchDimmerGP(TuyaDimmerSwitch):
         MODELS_INFO: [
             ("_TZE200_3p5ydos3", "TS0601"),
             ("_TZE200_ip2akl4w", "TS0601"),
+            ("_TZE200_vucankjx", "TS0601"),  # Loratap
```

Comment:
Dont adding this one more time in this PR then you have doing it in the original and also the same for line 211.

Reply:
I accidentally pushed the wrong file, latest is the correct one.